README.md

```diff
@@ -60,17 +34,18 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Microverse Mini Project]
+
+> A project to demonstrate good coding practices.
 
-> Describe your project in 1 or 2 sentences.
 
-**[your_project__name]** is a...
+**Microverse Mini Project** is built with HTML and CSS
 
-## 🛠 Built With <a name="built-with"></a>
+## 🛠 Built With HTML and CSS <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
+> This project is to show proper structure to be followed when in a professional work setting.
 
 <details>
   <summary>Client</summary>
```

Comment:
- [ ] Kindly add the relevant `tech-stacks` like as of now you can consider `HTML` and `CSS` in your `client` sub-section because you are not using `React.js`, `Express.js` and `PostgreSQL`. It makes your repo looks unprofessional.